libs/SmartSync/SmartSyncTests/SyncManagerTests.m

```diff
@@ -976,7 +1055,7 @@ - (NSString*) createLocalId
 
 - (void) deleteSyncs
 {
-    [store clearSoup:ACCOUNTS_SOUP];
+    [store clearSoup:kSFSyncStateSyncsSoupName];
```

Comment:
Syncs were not cleaned up between tests because the clear was going to the wrong soup.